src/components/ScholarshipForm.jsx

```diff
@@ -128,6 +131,40 @@ function ScholarshipForm({ scholarship }) {
         'Include information that is required for applicants to have.',
       content: (
         <Grid container spacing={3}>
+          <Grid item xs={12}>
+            {openAlert && lintIssues?.messages?.length > 0 && (
+              <Alert
+                severity="warning"
+                sx={{ mt: 2 }}
+                onClose={() => {
+                  setOpenAlert(false);
+                }}>
+                <AlertTitle>
+                  <strong>
+                    We found the following potential requirements in the
+                    description. Would you like to populate these values?
+                  </strong>
+                </AlertTitle>
+                <Box component="ul">
+                  {lintIssues.messages?.map((m, i) => (
+                    <Typography key={i} component="li">
+                      {m}
+                    </Typography>
+                  ))}
+                </Box>
+                <Button
+                  onClick={() => {
+                    const vals = formik.values.requirements;
+                    formik.setFieldValue('requirements', {
+                      ...vals,
+                      ...lintIssues.reqs,
+                    });
```

Comment:
Probably want to do something like formik.setFieldValue('requirements.X', [...reqs.X, lintReqs.X]) for each requirement. Or change this to do that. E.g.:
```suggestion
                    const vals = formik.values.requirements;
                    formik.setFieldValue('requirements', {
                      ...vals,
                      requirements = {
                        grades: [...reqs.grades, ...lintReqs.grades], 
                        ...
                      },
                    });
```